nixos/modules/services/web-servers/phpfpm/default.nix

```diff
@@ -175,16 +184,7 @@ in {
         '';
       };
 
-      phpPackage = mkOption {
```

Comment:
This option is used on line 66 and 67. I do not want this option removed as I use it very often.

Reply:
https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/services/web-servers/phpfpm/default.nix#L66-L67 - here?

Reply:
Up 3 lines.
```
          default = cfg.phpPackage;
          defaultText = "config.services.phpfpm.phpPackage";
```

Reply:
replace to ?
```
          default = pkgs.php;
          defaultText = "pkgs.php";
```

Reply:
I **use** this option on **many** machines managed by nixops. I really don't want to see this option disappear as it is very useful to me. This allows me to override `php` version per machine on machines which have many `phpfpm` pools declared elsewhere.

Reply:
ok, removed commit